code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -39,23 +39,43 @@
 	return ..()
 
 /datum/reagent/slimejelly
+	data = list("colour"="#0b8f70")
```

Comment:
looking at the implementation. This data here just seems like it's a shadow copy of color.
If you change this one you'll change color. They will at all times be the same (unless you change it and the `on_update` has not gone of yet)

Reply:
All data removed